### DISCUSSION TITLE
lost blank space between metric and data_time after update mmengine to 0.7.0

### DISCUSSION THREAD

**Discussion by r1cheu** [open-mmlab/mmengine#1016] [Q&A] [answered]
Hi, I'm using mmengine build my custom tool box. It works fine in mmengine v0.6.0. But after I update it to 0.7.0. data_time was add to metric log, but there is no blank sapce before it. where I should modify to make it proper?
log is below.
```python
03/24 22:41:18 - mmengine - INFO - Epoch(val) [55][8/8]  Corr/pearsonr: 0.8109  Corr/r2: 0.6548data_time: 0.0031  time: 0.0450  
```

**Comment by HAOCHENYE** [comment, marked as answer]
Thanks for your feedback. We'll fix this issue in [PR](https://github.com/open-mmlab/mmengine/pull/1025).
